README.md

```diff
@@ -90,11 +90,12 @@ and the JSONata expression `Phone.{type: number}`, an object constructor, the ac
 ```Split Property``` - use this field to choose a separator.
 
 ### Re-assemble Messages 
-**(Beta)**
 
-Inverse of the split action: Given a stream of incoming messages that which have 
-been split apart by a split action (or similar), produce one message once all 
-message parts have arrived.
+Inverse of the split action: Given a stream of incoming messages a sum message is generated.
+Has 3 different behaviour variants:
```

Comment:
Can I suggest that we add a dropdown to the `config` where a user would select one of the three modes? Then based on the selected mode, the appropriate inputs would be shown. (Then we also gain the ability to provide a default value for the timeout.)

Reply:
Thank you for the suggestion. I have implemented it.